build.gradle

```diff
@@ -60,6 +60,10 @@ dependencies {
     implementation group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.7.0'
     implementation group: 'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: '2.7.4'
 
+    implementation 'com.konghq:unirest-java:3.11.09'
+
+    implementation 'org.json:json:20220924'
+
```

Comment:
Not sure if we still need `org.json:json` package, since we already had `com.fasterxml.jackson.core`